src/app/info/InfoPage.tsx

```diff
@@ -30,8 +30,8 @@ const InfoPage: React.FC<QueryType> = ({ data, error }) => {
         console.error(error);
         return <h3>Error occured when fetching data.</h3>;
     } else {
-        const topRow = data;
-        return <WikiItem row={topRow} />;
+        const rows: DbWikiRow[] = data;
+        return <WikiItems rows={rows} />;
     }
```

Comment:
In general it's best practice to only have one return statement in a react component (happy to chat about why if you want). I think I'd prefer something like:
```
return ( 
    <>
        { error ? 
            <h3>...</h3>
            : <WikiItems rows ={rows} />
        }
    </>
)
```
I think that should work with types? Let me know if there's any problems